pom.xml

```diff
@@ -612,20 +570,28 @@
 
         <profile>
             <id>hbase-1</id>
-            <!-- Empty profile to prevent warning when compiling with -Phbase-1 -->
-        </profile>
-
-        <profile>
-            <id>hbase-2</id>
-            <properties>
-                <shims.artifactId>${shims2.artifactId}</shims.artifactId>
-                <shims.module>hbase-2</shims.module>
-                <java.version>1.8</java.version>
-                <hadoop.version>${hadoop2.version}</hadoop.version>
-                <hbase.version>${hbase2.version}</hbase.version>
-                <hbase.artifactId.suffix>${hbase2.artifactId.suffix}</hbase.artifactId.suffix>
-                <commons-io.version>2.11.0</commons-io.version>
-            </properties>
+              <build>
+                <plugins>
+                  <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-enforcer-plugin</artifactId>
+                    <executions>
+                      <execution>
+                        <id>enforce</id>
+                        <goals>
+                          <goal>enforce</goal>
+                        </goals>
+                        <configuration>
+                          <rules>
+                            <AlwaysFail/>
```

Comment:
Even if this is just a draft, why AlwaysFail?

Reply:
I've meant this as a courtesy to let people who try to update to 1.1 without checking the release notes know early that it won't work.
If I removed this, the old build command for HBase 1.x would sill work, even though HBase 1.x is no longer supported.

It is not essential, as we've also changed the artifact names, so they would figure out when the expected artifacts are not present at the latest.

Reply:
We can add the same AlwaysFail for hbase-2 too, or just remove both profiles, I'm fine either way.